src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,35 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager extends Employee {
+
+    static List<String> managers = new ArrayList<>();
+    Manager(String name, double salary){
+        super(name,salary);
+        managers.add(name);
+    }
+
+   boolean isHire(String name, double salary)
```

Comment:
Ciało tej metody dotyczy stricte `EmployeeNumberManager`a - powinno się znajdować w nim. Tutaj zaś miała być jedynie deklaracja tej metody:
`public abstract boolean isHire(Employee employee)` - zauważ, że zmieniłem ją tak by przyjmowala jako parametr pracownika, a nie jego pensje i imię, tak jest to dużo czytelniejsze i bardziej "obiektowe" :)